package.json

```diff
@@ -4,12 +4,12 @@
   "description": "LeapDAO bridge contracts (Plasma + PoS)",
   "main": "truffle-config.js",
   "scripts": {
-    "test": "truffle compile && scripts/test.sh",
+    "test": "scripts/test.sh",
```

Comment:
why no yarn compile? wouldn't the test run without recently added changes?

Reply:
I've noticed that `truffle test` (it is called inside `test.sh`) also does compile. So it is double compile, hence this PR

Reply:
Example. See here https://travis-ci.org/leapdao/leap-contracts/jobs/611872926

Line 235 we are calling `truffle compile`. Line 412 it is being called by `truffle test` (does nothing, still consumes some time)

Reply:
removing this wasn't the best decision: for some reason `truffle test` is not caching the compilation results, but uses `truffle compile` cache. So if you change the contracts and run tests multiple times it will compile the changes every time, very annoying..

Reply:
reverting here https://github.com/leapdao/leap-contracts/pull/268